pulsar-client-cpp/include/pulsar/c/producer_configuration.h

```diff
@@ -43,6 +43,24 @@ typedef enum {
     pulsar_CompressionZLib = 2
 } pulsar_compression_type;
 
+typedef enum {
```

Comment:
I have a recent change at #3940 that make the schema type aligned across multiple places. Can you make sure the enum number also aligned with the java one?